src/cmd/windows/powershellhelper.ps1

```diff
@@ -0,0 +1,85 @@
+# By default PowerShell adds a byte order mark (BOM) to the beginning of a file when using Out-File with a unicode encoding such as UTF8.
+# This causes the Jenkins output to contain bogus characters because Java does not handle the BOM characters by default.
+# This code mimics Out-File, but does not write a BOM.  Hopefully PowerShell will provide a non-BOM option for writing files in future releases.
```

Comment:
https://stackoverflow.com/questions/5596982/using-powershell-to-write-a-file-in-utf-8-without-the-bom :scream:

Reply:
One pleasant surprise is that `go` only supports `UTF8` with no BOM, so `powershellhelper.ps1` or any other powershell tricks are not needed in the binary version.

Reply:
removed powershellhelper.ps1 in fcdcaa6701b051b207ed025ffea01ba51d48364e